### ISSUE TITLE
Hi,how can i get the ted-59 and opus-100 dataset?

### ISSUE THREAD

**Issue by altctrl00** [cordercorder/nmt-multi#1]

**Comment by cordercorder** [comment]
Hi, thanks for your attention. 

The TED-59 dataset can be accessed from [http://phontron.com/data/ted_talks.tar.gz](http://phontron.com/data/ted_talks.tar.gz). The python script `ted_reader.py` in this [repository](https://github.com/neulab/word-embeddings-for-nmt) can be used to read the parallel corpus.

The OPUS-100 dataset can be found at [https://github.com/EdinburghNLP/opus-100-corpus](https://github.com/EdinburghNLP/opus-100-corpus).

**Comment by altctrl00** [comment]
Thanks,btw,your work is very exciting!

**Comment by cordercorder** [comment]
Thank you. I am really excited that you like our work.

**Comment by altctrl00** [comment]
it seems that the ted dataset you mentioned has 60 languages ,but you only use 59?

**Comment by cordercorder** [comment]
Yes, it seems that there are 60 language pairs in the raw dataset. But the column named `calv` is empty, which indicates that there is no parallel corpus between `calv` and `en`. Therefore, there are 59 language pairs in the TED-59 dataset.
